src/EVEMon.Common/Data/MasteryShip.cs

```diff
@@ -91,12 +91,12 @@ internal void Initialize()
             while (true)
             {
                 bool updatedAnything = Items
-                    .Aggregate(false, (current, mastery) => current | mastery.TryUpdateMasteryStatus());
+                    .Aggregate(false, (current, mastery) => current || mastery.TryUpdateMasteryStatus());
```

Comment:
Actually this one could cause problems, the method call updates stuff.

Reply:
`TryUpdateMasteryStatus` does update the mastery's listed level so leaving it out entirely will be bad (the mastery level displayed might be incorrect). The main killer is `prereqSkill.AllDependencies.All` which is invoked in the method, and this is invoked for every prerequisite skill even if they have already been queried. I am making a chance to remember this value (it is only a boolean per skill) to hopefully cut the load time.